submissions/adeyemiJuwon/dom/product.js

```diff
@@ -0,0 +1,73 @@
+let quantity = document.getElementById("quantity");
+let price = document.getElementById("price");
+let shippingCost = document.getElementById("shippingCost");
+let totalCost = document.getElementById("totalCost");
+let button = document.getElementById("btn");
+let customer = document.getElementById('customer-price');
+let grossPrice = quantity.value * Number(price.innerHTML);
+
+
+
+function shipp(){
+	grossPrice = quantity.value * Number(price.innerHTML);
+	if ( grossPrice < 1000){
+		shipping = 100
+		shippingCost.innerHTML = "£" + shipping + " GBP";
+	}
+	else if(grossPrice >= 1000 && grossPrice <= 20000){
+		shipping= 0.15 * grossPrice
+		shippingCost.innerHTML = "£" + shipping + " GBP";
+	}else if (grossPrice >= 20000 && grossPrice <= 100000){
+		shipping= 0.20 * grossPrice
+		shippingCost.innerHTML = "£" + shipping + " GBP";
+	}else{
+		shipping = 25000
+		shippingCost.innerHTML = "£" + shipping + " GBP";
+	}
+	return true
+}
+
+
+
+function cost(){
+	shipp();
```

Comment:
Calling the ship function does not make shipping variable and its value available. You might want to consider joining the 2 functions. Also, when you make changes you want to test that your code does what it suppose to do. @Samju200

Reply:
Yes, but the changes still work perfectly with what i want . declaring shipping variable in the function inside the cost function will not work expect i join the two function like you suggest. but i thought calling the function inside the another make every information declare in the  the function available to the function.